digits/dataset/tasks/create_db.py

```diff
@@ -91,13 +93,15 @@ def __setstate__(self, state):
                     self.encoding = 'none'
                 delattr(self, 'encode')
             else:
-                self.encoding = 'none'
+                self.encoding = 'none' 
         self.pickver_task_createdb = PICKLE_VERSION
 
         if not hasattr(self, 'backend') or self.backend is None:
             self.backend = 'lmdb'
         if not hasattr(self, 'compression') or self.compression is None:
             self.compression = 'none'
+        if not hasattr(self, 'augmentation'):
+            self.augmentation = None
```

Comment:
maybe you need to do:
`self.augmentation = {}`
I had an error when loading an older dataset:
```
  File "/fast-scratch/gheinrich/ws/digits/digits/templates/datasets/images/classification/show.html", line 105, in template
    {% if task.augmentation|length != 0 %}
TypeError: object of type 'NoneType' has no len()
```
I think this is because self.augmentation is set to None here.